tests/docker/start.globus.run.tests.el.sh

```diff
@@ -0,0 +1,87 @@
+#! /bin/bash
+
+# install iRODS client packages built by development environment 
+/install_local_irods_packages.sh
```

Comment:
Pardon, I've not done bash in a while, is this the best way to run another script?

Reply:
Seems fine to me.

Do you feel the line in question is problematic?

Reply:
Nope, just needed a bash refresher. Going to resolve.